google-cloud-core/lib/google/cloud.rb

```diff
@@ -90,36 +90,31 @@ def self.new project_id = nil, credentials = nil, retries: nil, timeout: nil
     #     Google::Cloud libraries.
     #
     def self.configure
+      @config ||= Config.create
```

Comment:
Can we just call this once during the class definition, or are you concerned about the file being required multiple times?

Reply:
Its more about the variable not being used by anything other than the method, and a desire to have the variable "contained" to the method. The `configure` method isn't the inner-most loop so I don't sweat the extra nil check in regards to performance.